src/components/Compare/Compare.style.tsx

```diff
@@ -359,21 +354,40 @@ export const DataCellValue = styled.span<{
   text-transform: capitalize;
 `;
 
+export const VaccinationsCell = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
nit- flex-direction defaults to row so no need to declare it

Reply:
oops, good catch!